src/praisonai-code/praisonai_code/cli/features/capabilities.py

```
@@ -1234,7 +1234,33 @@ def handle_responses(args, unknown_args):
+        if any(arg in retrieval_flags for arg in unknown_args):
+            try:
+                from ..commands.retrieval import register_commands
+            except ImportError as e:
+                print(f"ERROR: {e}")
+                return 1
+            import typer as _typer
+
+            retrieval_app = _typer.Typer()
+            register_commands(retrieval_app)
+            try:
+                retrieval_app(["search"] + list(unknown_args), standalone_mode=False)
```

Ruff (0.15.20)

[warning] Consider `["search", *list(unknown_args)]` instead of concatenation

Replace with `["search", *list(unknown_args)]`

(RUF005)